manager/src/views.rs

```diff
@@ -71,4 +84,107 @@ impl Contract {
         let task = self.tasks.get(&task_hash).expect("No task found by hash");
         task
     }
+
+    /// Gets amount of tasks alotted for a single agent per slot
+    ///
+    /// ```bash
+    /// near view cron.testnet get_total_tasks_per_agent_per_slot
+    /// ```
+    pub fn get_total_tasks_per_agent_per_slot(&self) -> u64 {
+        // assess if the task ratio would support a new agent
+        let [agent_ratio, task_ratio] = self.agent_task_ratio;
+
+        // Math example:
+        // ratio [2 agents, 5 tasks]
+        // agent can execute 5 tasks per slot
+        task_ratio.div_euclid(agent_ratio)
+    }
+}
+
+#[cfg(all(test, not(target_arch = "wasm32")))]
```

Comment:
as far as I know, all we need is:
`#[cfg(test)]`
but I see people using a line similar to this a lot. more of a "the more you know" thing to share

Reply:
interesting